sdk/storage/azure-storage-queues/inc/azure/storage/queues.hpp

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include "azure/storage/common/storage_exception.hpp"
```

Comment:
Same issue here, and possibly elsewhere. Keep queues.hpp simplified header self-contained for headers in that package only.

Reply:
duplicate, will discuss in another thread